frontend/src/modules/history/components/Translation.js

```diff
@@ -121,6 +126,21 @@ export default class Translation extends React.Component<Props> {
                     />
                 </div>
                 <menu className="toolbar">
+                { translation.rejected ?
+                    // Delete Button
+                    <Localized
+                        id="history-translation-button-delete"
+                        attrs={{ title: true }}
+                    >
+                        <button
+                            className='delete far'
+                            title='Delete'
+                            onClick={ this.delete }
+                        />
+                    </Localized>
+                    :
+                    null
```

Comment:
I like to reverse those conditions so that I can put the `? null :` on the first line, and make it overall a little bit more compact. It also puts the `:` operator closer to the condition, which I believe makes it easier to understand.